README.md

```diff
@@ -128,8 +141,17 @@ All props accept arrays as values for mobile-first responsive styles.
 
 styled-system includes a higher order component to add style props to any component.
 
+**With `styled-components`**
+
+```jsx
+import hoc from 'styled-system/dist/styled-components/hoc'
```

Comment:
That sounds fine!

> Although it'd still install deps

Actually `styled-system` doesn't currently have `styled-components` listed in `dependencies`. And I think that's fine, especially with this change :+1: